ui/component/common/form-components/form-field.jsx

```diff
@@ -101,7 +101,9 @@ export class FormField extends React.PureComponent<Props> {
       } else if (type === 'select') {
         input = (
           <fieldset-section>
-            {label && <label htmlFor={name}>{label}</label>}
+            {(label || errorMessage) && (
```

Comment:
error message replacing label seems a bit weird?

Reply:
I agree. It's mostly for inputs that are side by side with other inputs or buttons. Adding it to the top or bottom messes up the side by side right now in a number of cases.